flag.go

```diff
@@ -8,9 +8,9 @@ type Flag int
 
 const (
 	// Set for the last xrow in a transaction.
-	IPROTO_FLAG_COMMIT Flag = 0x01
+	IPROTO_FLAG_COMMIT Flag = 0
 	// Set for the last row of a tx residing in limbo.
-	IPROTO_FLAG_WAIT_SYNC Flag = 0x02
+	IPROTO_FLAG_WAIT_SYNC Flag = 1
 	// Set for the last row of a synchronous tx.
-	IPROTO_FLAG_WAIT_ACK Flag = 0x04
+	IPROTO_FLAG_WAIT_ACK Flag = 2
```

Comment:
Wrong values:
https://github.com/tarantool/tarantool/blob/05751e6c6869579446b34ae67e0605a68fc56b89/src/box/iproto_constants.h#L61

Reply:
Updated.

Reply:
Could you please to show the change in the code generator?